orx-syphon/src/test/kotlin/ClientExample.kt

```diff
@@ -0,0 +1,20 @@
+import org.openrndr.application
```

Comment:
src/test is intended for tests in a quite specific format, let's see if it trips up the build server :)